tslint.json

```diff
@@ -10,7 +10,8 @@
     "member-ordering": [false],
     "no-var-requires": false,
     "no-empty": false,
-    "no-console": false
+    "no-console": false,
+    "variable-name": ["allow-leading-underscore"]
```

Comment:
Add `"variable-name": ["allow-leading-underscore”]` due to a lack of the ability to prefix backing variables with underscore. This is an [ongoing discussion](https://github.com/palantir/tslint/issues/1489), and something the community thinks should be implemented, as it the preferred way to use a property with backing variable in TypeScript.

Reply:
Fine by me